tests/connectors/prometheus_test.py

```diff
@@ -582,15 +579,13 @@ async def burst_traffic() -> None:
                         )
 
                 connector = PrometheusConnector(config=config, optimizer=optimizer)
-                event_loop.call_later(15, asyncio.create_task, burst_traffic())
+                event_loop.call_later(30, asyncio.create_task, burst_traffic())
                 measurement = await asyncio.wait_for(
-                    connector.measure(control=servo.Control(duration="10m")),
-                    timeout=300,  # NOTE: if we haven't returned in 5 minutes all is lost
+                    connector.measure(control=servo.Control(duration="5m")),
+                    timeout=360,  # NOTE: if we haven't returned in 5 minutes all is lost
                 )
                 assert measurement
-                assert (
-                    len(measurement) == 1
-                ), "expected one TimeSeries (error_rate should be absent)"
+                assert len(measurement) == 1
```

Comment:
```suggestion
                assert len(measurement) == 1, devtools.pprint(measurement)
```
Need to `import devtools` at the top as well